Updated manifest.json

superoo7 opened the pull request:
[Reference on manifest.json](https://developers.google.com/web/fundamentals/web-app-manifest/)

untested

relativityboy commented:
@superoo7 I like this, but please see our [contribution guidelines](https://github.com/steemit/condenser/blob/master/CONTRIBUTING.md#enhancement-suggestions)

superoo7 commented:
# Issue
Since steemit has not have apps, so add to homescreen is necessary for a steemit user. No icon shown on my phone desktop when I add to homescreen.

_Reproducing this bug_
* Go to [steemit.com](https://www.steemit.com)
* Press add to homescreen on chrome (Android)

# Proposal
Modified the `manifest.json` by adding different size of images based on use screen size.

# Summary
[Reference on manifest.json](https://developers.google.com/web/fundamentals/web-app-manifest/)

What I added was a few modification to existing `manifest.json`

# Screenshots
![26178301_10210349159380580_188861830_o](https://user-images.githubusercontent.com/15358452/34393414-22eab3f0-eb8d-11e7-8278-93e08120b7ec.png)
_When Add to homescreen is pressed_

![26178301_10210349159380580_188861830_o](https://user-images.githubusercontent.com/15358452/34393414-22eab3f0-eb8d-11e7-8278-93e08120b7ec.png)
![26177549_10210349158900568_2061556319_o](https://user-images.githubusercontent.com/15358452/34393417-26fd8cba-eb8d-11e7-97bb-b93613a421a2.png)
_When it is added to homescreen_
